components/SettingsComponent.vue

```diff
@@ -44,17 +44,14 @@ const d = Object.fromEntries(Object.entries(data).filter(([k, v]) =>
 .badgePadding :deep(p) {
   margin: 4px;
 }
-.settingsList h3:first-of-type {
+.settingsList article:first-of-type {
   border-top-width: 0;
-  padding-top: 0;
 }
-.settingsList h3 {
+.settingsList article {
   border-top: 1px solid var(--vp-c-divider);
-  margin-top: 18px;
-  padding-top: 18px;
 }
-.settingsList .rightAlign {
-  text-align: right;
```

Comment:
In this commit you replace the `<th class="rightAlign">` with `<th style="text-align:right;">`; and in the next commit you replace this again with a proper styling rule.

You could also just combine both into this here commit.

Reply:
I think the merging left some nasty conflicts, so looks like I got confused when trying to break apart again.  For interests of time, I will just smush the two together.